cmd/fleetctl/get.go

```diff
@@ -737,16 +737,22 @@ func getCarvesCommand() *cli.Command {
 					completion = "Expired"
 				}
 
+				errored := "no"
+				if c.Error != nil {
+					errored = "yes"
+				}
+
 				data = append(data, []string{
 					strconv.FormatInt(c.ID, 10),
-					c.CreatedAt.Local().String(),
+					c.CreatedAt.String(),
```

Comment:
this was the only place were we used local time instead of UTC, so I changed it for consistency.